src/pages/promo-code/promo-code-edit/test/promo-code-edit.spec.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { useMutation, useQuery } from '@apollo/client';
+
+import PromoCodeEdit from '../promo-code-edit';
+import { mocks } from './promo-code-edit.variables';
+
+jest.mock('react-redux');
+const dispatch = jest.fn();
+useDispatch.mockImplementation(() => dispatch);
+
+jest.mock('@apollo/client');
```

Comment:
We should not mock apollo/client.

https://www.apollographql.com/docs/react/development-testing/testing/

Reply:
done